src/excmds.ts

```diff
@@ -4302,8 +4300,12 @@ const KILL_STACK: Element[] = []
         - -q* quick (or rapid) hints mode. Stay in hint mode until you press <Esc>, e.g. `:hint -qb` to open multiple hints in the background or `:hint -qW excmd` to execute excmd once for each hint. This will return an array containing all elements or the result of executed functions (e.g. `hint -qpipe a href` will return an array of links).
         - -J* disable javascript hints. Don't generate hints related to javascript events. This is particularly useful when used with the `-c` option when you want to generate only hints for the specified css selectors. Also useful on sites with plenty of useless javascript elements such as google.com
           - For example, use `bind ;jg hint -Jc .rc > .r > a` on google.com to generate hints only for clickable search results of a given query
-        - -br deprecated, use `-qb` instead
         - -F [callback] - run a custom callback on the selected hint, e.g. `hint -JF e => {tri.excmds.tabopen("-b",e.href); e.remove()}`.
+        - -V create hints for invisible elements. By default, elements outside the viewport when calling :hint are not hinted, this includes them anyways.
+        - -! perform action immediately. This effectively selects every hinted element in sequence
+          - For example, `hint -!bf Comments` opens in background tabs all visible links whose text matches `Comments`
+        - [selector] hint links that match the css selector
+          - `bind ;c hint [class*="expand"],[class="togg"]` works particularly well on reddit and HN
```

Comment:
We should mention the deprecated -c bind because it's been around for so many years that people will bump into it in the wild and wonder what it did

Reply:
Thinking back on it I actually didn't need to remove the -c flag, I just need to consider it's another argument with an option and push the corresponding value to `positionals`.